examples/manila-csi-plugin/nfs/snapshot/snapshotrestore.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   dataSource:
     name: new-nfs-share-snap
     kind: VolumeSnapshot
-    apiGroup: snapshot.storage.k8s.io
+    apiGroup: snapshot.storage.k8s.io/1beta1
```

Comment:
not v1beta1?

Reply:
`apiGroup` field expects a non-versioned API name after all. Fixed in https://github.com/kubernetes/cloud-provider-openstack/pull/1205/commits/e992e0520b12e48432e2323808e04b396815c590 .